config/routes.rb

```diff
@@ -10,25 +10,14 @@
       end
     end
 
-    resources :products, :only => [] do
-      member do
-        get :available_repositories
-        put :toggle_repository
-      end
-    end
-
-    resources :providers, :only => [] do
-      collection do
-        get :redhat_provider
-        get :redhat_provider_tab
-      end
-    end
-
     if Katello.with_remote_execution?
       match '/remote_execution' => 'remote_execution#create', :via => [:post]
     end
   end
 
+  get '/katello/providers/redhat_provider', to: redirect('/subscriptions')
```

Comment:
Is was expecting redirect from old RH repos page (redhat_provider) to new RH repos page (/redhat_repositories)

Reply:
Lol, oops, too many things going on.  Apparently the RH subscriptions page is on my mind.  Will fix.